src/board.py

```diff
@@ -131,9 +133,9 @@ def __repr__(self) -> str:
         return (
             "\n▐\033[7m  A B C D E F G H  \033[0m▌\n" +
             "\n".join(
-                f"▐\033[7m{self.index_to_rank[index]}\033[0m▌" +
-                " ".join(str(piece) for piece in rank) +
-                f"▐\033[7m{self.index_to_rank[index]}\033[0m▌"
+                f"▐\033[7m{self.index_to_rank[index]}\033[27m\033[4m▌" +
+                "│".join(str(piece) for piece in rank) +
+                f"▐\033[24m\033[7m{self.index_to_rank[index]}\033[0m▌"
```

Comment:
This changes the representation of the board. A unit test expects a specific representation and will fail after this change. Please update the unit test.

Reply:
Very well.